### PR TITLE
added login stuff

### PR DESCRIPTION
**Pull request by mikeya**

### CODE REVIEW

File: src/screens/Login.js [hunk 1]
```diff
@@ -28,11 +28,21 @@ export default class Login extends Component {
...
+  
+
 	login = async () => {
         try {
-            const response = await login(this.state.email, this.state.password)
-            await AsyncStorage.setItem("userToken", response.token)
-        } catch(e ){
+            const response = await login(this.state.email, this.state.password);
+            await AsyncStorage.setItem("userToken", response.token);
+            this.props.navigation.navigate('AppStack');
```
Comment (joshgarza) [inline src/screens/Login.js:44]: I remember you mentioning something about this AppStack navigation flow last night. Can you explain what it is to me?

File: ios/trainer.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -874,7 +874,7 @@
 					00E356ED1AD99517003FC87E = {
 						CreatedOnToolsVersion = 6.2;
 						DevelopmentTeam = 29DWXCY7LN;
-						ProvisioningStyle = Manual;
+						ProvisioningStyle = Automatic;
```
Comment (joshgarza) [inline ios/trainer.xcodeproj/project.pbxproj:877]: Can you give me the quick and dirty on what the changes in this file are?

File: ios/trainer/Info.plist [hunk 3]
```diff
@@ -2,6 +2,14 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>NSExceptionDomains</key>
```
Comment (joshgarza) [inline ios/trainer/Info.plist:5]: Can you give me a high level explanation of the changes in this file?

**Review by joshgarza — APPROVED** [review]
This was fun. All my questions come from my own ignorance and wanting to learn a bit more about how these changes work.

File: src/screens/Login.js (hunk 1 above)
Comment (mikeya) [reply src/screens/Login.js:44]: Followed the advice here: https://reactnavigation.org/docs/en/auth-flow.html

Basically theres two navigation stacks, Auth and App, they cannot talk to each other. In AuthLoader we're checking for a userToken, if it exists we send to the AppStack which for now just has the Dashboard (see Router.js) else we send to the AuthStack

File: ios/trainer.xcodeproj/project.pbxproj (hunk 2 above)
Comment (mikeya) [reply ios/trainer.xcodeproj/project.pbxproj:877]: Basically we need to get a provisioning certificate to our app builds. a provisioning certificate basically says "yo this app is able to be installed on this device for blah reason" in this case we're switching I believe the dev build to use the dev provisioning license, and the release build to use the ad hoc provisioning license, which basically allows us to distribute our app to multiple test devices (think beta users), I'll add you to appCenter and you can download a release version of the build, you'll notice it's snappier and you can't develop with it

File: ios/trainer/Info.plist (hunk 3 above)
Comment (mikeya) [reply ios/trainer/Info.plist:5]: this is basically our main config file for our ios app. we're allowing insecure connections from localhost here, so we can hit http://localhost:5000 for local testing, as opposed to https:

**Pull request merged**